File: 10929/Main.java
import java.io.*;

public class Main {
   public static void main(String args[]) throws IOException {
      String line;
      String answer = "";
      BufferedReader in = new BufferedReader(new InputStreamReader(System.in));

      while ((line = in.readLine()) != null) {
         if (line.charAt(0) == '0' && line.length() == 1)
            break;
         int sum_odd = 0, sum_even = 0;
         for (int i = 0; i < line.length(); i++) {
            if ((i % 2) != 0) {
               sum_odd += (line.charAt(i) - '0');
            } else {
               sum_even += (line.charAt(i) - '0');
            }
         }
         answer += (line + " is" + ((sum_odd - sum_even) % 11 == 0 ? " a multiple of 11.\n" : " not a multiple of 11.\n"));
      }
      System.out.print(answer);
   }
}

File: 10929/main.c
#include <stdio.h>

int main(int argc, char* argv[])
{
   char input[1001];
   int i, sum_odd, sum_even;

   while (gets(input) != NULL) {
      if (input[0] == '0' && input[1] == '\0')
         break;
      sum_odd = sum_even = 0;
      for (i = 0; input[i] != '\0'; i++) {
         if (i & 1) {
            sum_odd += (input[i] - '0');
         } else {
            sum_even += (input[i] - '0');
         }
      }
      printf("%s is%sa multiple of 11.\n", input, ((sum_odd - sum_even) % 11 == 0 ? " " : " not "));
   }

   return 0;
}
